diff --git a/node_modules/ts-pdf/dist/ts-pdf.d.ts b/node_modules/ts-pdf/dist/ts-pdf.d.ts
index 9c10e60..aa6d4ec 100644
--- a/node_modules/ts-pdf/dist/ts-pdf.d.ts
+++ b/node_modules/ts-pdf/dist/ts-pdf.d.ts
@@ -951,7 +951,7 @@ declare type BaseFileButtons = "open" | "close";
 declare type FileButtons = BaseFileButtons | "save";
 interface TsPdfViewerOptions {
     containerSelector: string;
-    workerSource: string;
+    workerSource: any;
     userName?: string;
     annotChangeCallback?: (detail: AnnotEventDetail) => void;
     customStamps?: CustomStampCreationInfo[];
diff --git a/node_modules/ts-pdf/dist/ts-pdf.esm.js b/node_modules/ts-pdf/dist/ts-pdf.esm.js
index 4a9b3aa..a046ba6 100644
--- a/node_modules/ts-pdf/dist/ts-pdf.esm.js
+++ b/node_modules/ts-pdf/dist/ts-pdf.esm.js
@@ -66,23 +66,20 @@ const mainHtml = `
         </div> 
       </div>
       <div id="modes" class="subpanel panel-item">
-        <div id="button-mode-text" class="panel-button panel-item">
+        <div id="button-mode-text" class="panel-button panel-item" title="Text Tool">
           <img src="${Icons.icon_caret}"/>
         </div> 
-        <div id="button-mode-hand" class="panel-button panel-item">
+        <div id="button-mode-hand" class="panel-button panel-item" title="Grab Tool">
           <img src="${Icons.icon_hand}"/>
         </div> 
-        <div id="button-mode-annotation" class="panel-button panel-item">
+        <div id="button-mode-annotation" class="panel-button panel-item" title="Annotation Tool">
           <img src="${Icons.icon_popup}"/>
         </div> 
-        <div id="button-mode-comparison" class="panel-button panel-item">
-          <img src="${Icons.icon_compare}"/>
-        </div> 
         <div class="panel-v-separator margin-s-5 panel-item"></div>
-        <div id="button-open-file" class="panel-button panel-item">
+        <div id="button-open-file" class="panel-button panel-item" title="Open PDF">
           <img src="${Icons.icon_load}"/>
         </div> 
-        <div id="button-save-file" class="panel-button panel-item">
+        <div id="button-save-file" class="panel-button panel-item" title="Save PDF...">
           <img src="${Icons.icon_download}"/>
         </div> 
         <div id="button-close-file" class="panel-button panel-item">
@@ -92,10 +89,10 @@ const mainHtml = `
     </div>
     <div id="bottom-panel">
       <div id="paginator" class="subpanel panel-item">
-        <div id="paginator-prev" class="panel-button">
+        <div id="paginator-prev" class="panel-button" title="Previous Page">
           <img src="${Icons.icon_arrow_up}"/>
         </div>
-        <div id="paginator-next" class="panel-button">
+        <div id="paginator-next" class="panel-button" title="Next Page">
           <img src="${Icons.icon_arrow_down}"/>
         </div>
         <input id="paginator-input" type="text">
@@ -104,25 +101,25 @@ const mainHtml = `
       </div>
       <div class="panel-v-separator panel-item"></div>
       <div id="rotator" class="subpanel panel-item">
-        <div id="rotate-counter-clockwise" class="panel-button">
+        <div id="rotate-counter-clockwise" class="panel-button" title="Rotate Counter-Clockwise">
           <img src="${Icons.icon_counter_clockwise}"/>
         </div>
-        <div id="rotate-clockwise" class="panel-button">
+        <div id="rotate-clockwise" class="panel-button" title="Rotate Clockwise">
           <img src="${Icons.icon_clockwise}"/>
         </div>
       </div>
       <div class="panel-v-separator panel-item"></div>
       <div id="zoomer" class="subpanel panel-item">
-        <div id="zoom-out" class="panel-button">
+        <div id="zoom-out" class="panel-button" title="Zoom Out">
           <img src="${Icons.icon_minus}"/>
         </div>
-        <div id="zoom-in" class="panel-button">
+        <div id="zoom-in" class="panel-button" title="Zoom In">
           <img src="${Icons.icon_plus}"/>
         </div>
-        <div id="zoom-fit-viewer" class="panel-button">
+        <div id="zoom-fit-viewer" class="panel-button" title="Fit to View">
           <img src="${Icons.icon_fit_viewer}"/>
         </div>
-        <div id="zoom-fit-page" class="panel-button">
+        <div id="zoom-fit-page" class="panel-button" title="Fit to Page">
           <img src="${Icons.icon_fit_page}"/>
         </div>
       </div>
@@ -146,87 +143,69 @@ const mainHtml = `
     <div id="annotation-panel">
       <div class="annotation-panel-row">
         <div id="button-annotation-edit-text" 
-          class="panel-button annotation-panel-subitem">
+          class="panel-button annotation-panel-subitem" title="Edit Annotation">
           <img src="${Icons.icon_text}"/>
         </div> 
         <div id="button-annotation-delete" 
-          class="panel-button annotation-panel-subitem">
+          class="panel-button annotation-panel-subitem" title="Delete Annotation">
           <img src="${Icons.icon_close}"/>
         </div> 
         <div id="button-annotation-mode-select" 
-          class="panel-button annotation-panel-item">
+          class="panel-button annotation-panel-item" title="Select Annotations">
           <img src="${Icons.icon_pointer}"/>
         </div> 
       </div>
-      <div class="annotation-panel-row">
-        <div id="button-annotation-stamp-undo" 
-          class="panel-button annotation-panel-subitem button-annotation-undo">
-          <img src="${Icons.icon_back}"/>
-        </div> 
-        <div id="button-annotation-stamp-clear" 
-          class="panel-button annotation-panel-subitem button-annotation-clear">
-          <img src="${Icons.icon_close}"/>
-        </div>
-        <div id="button-annotation-stamp-save" 
-          class="panel-button annotation-panel-subitem button-annotation-save">
-          <img src="${Icons.icon_ok}"/>
-        </div> 
-        <div id="button-annotation-mode-stamp" 
-          class="panel-button annotation-panel-item">
-          <img src="${Icons.icon_stamp}"/>
-        </div> 
-      </div>
       <div class="annotation-panel-row">
         <div id="button-annotation-pen-undo" 
-          class="panel-button annotation-panel-subitem button-annotation-undo">
+          class="panel-button annotation-panel-subitem button-annotation-undo" title="Undo">
           <img src="${Icons.icon_back}"/>
         </div> 
         <div id="button-annotation-pen-clear" 
-          class="panel-button annotation-panel-subitem button-annotation-clear">
+          class="panel-button annotation-panel-subitem button-annotation-clear" title="Clear">
           <img src="${Icons.icon_close}"/>
         </div> 
         <div id="button-annotation-pen-save" 
-          class="panel-button annotation-panel-subitem button-annotation-save">
+          class="panel-button annotation-panel-subitem button-annotation-save" title="Save">
           <img src="${Icons.icon_ok}"/>
         </div> 
         <div id="button-annotation-mode-pen" 
-          class="panel-button annotation-panel-item">
+          class="panel-button annotation-panel-item" title="Create/Edit Lines">
           <img src="${Icons.icon_pen}"/>
         </div> 
       </div>
       <div class="annotation-panel-row">
         <div id="button-annotation-geometric-undo" 
-          class="panel-button annotation-panel-subitem button-annotation-undo">
+          class="panel-button annotation-panel-subitem button-annotation-undo" title="Undo">
           <img src="${Icons.icon_back}"/>
         </div> 
         <div id="button-annotation-geometric-clear" 
-          class="panel-button annotation-panel-subitem button-annotation-clear">
+          class="panel-button annotation-panel-subitem button-annotation-clear" title="Clear">
           <img src="${Icons.icon_close}"/>
         </div> 
         <div id="button-annotation-geometric-save" 
-          class="panel-button annotation-panel-subitem button-annotation-save">
+          class="panel-button annotation-panel-subitem button-annotation-save" title="Save">
           <img src="${Icons.icon_ok}"/>
         </div> 
         <div id="button-annotation-mode-geometric" 
-          class="panel-button annotation-panel-item">
+          class="panel-button annotation-panel-item" title="Create/Edit Shapes">
           <img src="${Icons.icon_geometric}"/>
         </div>
       </div>
       <div class="annotation-panel-row">
         <div id="button-annotation-text-undo" 
-          class="panel-button annotation-panel-subitem button-annotation-undo">
+          class="panel-button annotation-panel-subitem button-annotation-undo" title="Undo">
           <img src="${Icons.icon_back}"/>
         </div> 
         <div id="button-annotation-text-clear" 
-          class="panel-button annotation-panel-subitem button-annotation-clear">
+          class="panel-button annotation-panel-subitem button-annotation-clear" title="Clear">
           <img src="${Icons.icon_close}"/>
         </div> 
         <div id="button-annotation-text-save" 
-          class="panel-button annotation-panel-subitem button-annotation-save">
+          class="panel-button annotation-panel-subitem button-annotation-save" title="Save">
           <img src="${Icons.icon_ok}"/>
         </div> 
         <div id="button-annotation-mode-text" 
-          class="panel-button annotation-panel-item">
+          class="panel-button annotation-panel-item" title="Create/Edit Annotation Text">
           <img src="${Icons.icon_text2}"/>
         </div>
       </div>
